api/alembic/versions/c92ba5b2b3d5_adds_audit_model.py

```diff
@@ -0,0 +1,41 @@
+"""Adds Audit model
+
+Revision ID: c92ba5b2b3d5
+Revises: 004ac48ffe18
+Create Date: 2021-04-12 14:33:00.507000
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'c92ba5b2b3d5'
+down_revision = '004ac48ffe18'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('audits',
```

Comment:
I'm inclined to say we should name this table something like `audit_user_login` because as we build out Preppy, we'll also need audit logs for modifications made to preparedness levels and the like, which I'd imagine will be in a separate table from this one. But I'd be interested to hear what @Sybrand has to say about this

Reply:
Auditing won't be normalized. So the create_date, create_user, update_date, create_user will be oin the revelevant table. E.g. if we have a "prep_level" table, and a user changes it, the audit columns will be in the prep_level table.
You make a good point re naming of the table though, though it's not audit_user_login, since the login happens on keycloak. A better name would tell us that these are audits relating to endpoints accessed by users. Don't know a short way to say that. "access_audits" ? "api_audits" ?

Reply:
"api_access_audits"?